src/dmd/glue.d

```diff
@@ -370,7 +372,6 @@ private void obj_end(Library library, const(char)* objfilename)
         //printf("write obj %s\n", objfilename);
         writeFile(Loc.initial, objfilename.toDString, objbuf[]);
     }
-    objbuf.dtor();
```

Comment:
This introduced a regression: https://issues.dlang.org/show_bug.cgi?id=22804

The buffer still needs to be cleared for the non-library case, e.g. with `objbuf.destroy()`.

Reply:
Sorry for the trouble. Are you already fixing it? I can do it if you wish :)

Reply:
No problem, please go ahead.

Reply:
Looking closer at this problem, perhaps partially reverting this patch solves the issue? https://github.com/dlang/dmd/commit/82d679fb5b7e58a4f8d6fbefae4022a153874512 . I can't reproduce your issue on Linux, although.

Reply:
Nevermind, you are right, the OutBuffer has a `~this()`.